docker-compose.yaml

```diff
@@ -0,0 +1,16 @@
+version: '3'
+
+services:
+  app:
+    image: aesirxio/concordium-sign-dapp:latest
+    environment:
+      REACT_APP_CLIENT_ID: ${REACT_APP_CLIENT_ID}
```

Comment:
Do we need all these variables here?  I think that instead we need variables to communicate with a JSON-RPC Concordium server

Reply:
I removed it @jatitoam , variables to communicate with JSON-RPC already in here https://github.com/vietredweb/concordium-sign-dapp/blob/main/src/config.ts

Reply:
We need to remove exposure of those variables in these public repos please.  They are private nodes.

Please move them to `.env` files.

Reply:
sure, I will move them to `.env`